### ISSUE TITLE
Soviet Cores

### ISSUE THREAD

**Issue by simsulla** [simsulla/nwomod#310]
**By JonInSpaec**
Here's something I bet you've never seen before! I enforced an Install Democracy CB during Operation Unthinkable...on the USSR...in 1948. This was, of course, awesome. However, it also had several curious effects. First, the Dissolution of the Soviet Bloc did not fire. All bloc members except Yugoslavia and Poland also installed democratic, right-leaning governments. This seems logical, but the Dissolution event seems as if it was not designed with an early Install Democracy CB in mind. Also, I understand the event chains relating to Berlin and Germany and such would still fire -- they refer by tag to Russia, not the USSR specifically. I'm not sure which of those would need to be changed for a democratic government type and the end of the Cold War. (The End of the Multipolar World event or whatever it is fired as well, as usual.) But the problem is, Hungary still rebels in 1956...eight years after Fidesz and Parnas came to power in my world. PARNAS-led Russia is then granted by event an Install Communism CB on Hungary. This doesn't make sense.
**By simsulla**
On the events probably the best thing is to introduce a Soviet core instead of using Russia. But that would require a lot of work.

**Comment by simsulla** [comment]
By @VictorWeikum in #264

Some crazy shit happened to me as well. While I played for Cuba (which I decided to stay democratic), Soviet Union renamed itself into Russia and became a democracy, but did not lose any territories, except for Baltic republics. They separeted about a half-year before. SU gone into war with Lithuania and the US to install communism on L. It did not end with SU turning into democratic Russia with liberal government (first half-year it was still communist, first elections changed it).

Before all that a number of countries stopped beign satallites of the SU. Now Bulgaria, East Germany and North Korea are satellites of Russia.

It all doesn't seem unreal (can't say about chances, I only played this version once), but I think that SU should've not renamed into Russia even becoming a democracy, if it doesn't get dissolved (need to make SU a country with cores or a communist name of Eurasian Union). And the war with L. should've stopped, of course.

Here are saves for analyse:
https://onedrive.live.com/redir?resid=54A943E8FE4717B8!76328&authkey=!AGlZNCp90sw4IzU&ithint=folder%2c
